language-extensions/fhir-extension/pom.xml

```diff
@@ -18,15 +18,15 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <asm.version>9.1</asm.version>
         <jsonschema2pojo-core.version>1.2.1</jsonschema2pojo-core.version>
-        <commons-lang3.version>3.14.0</commons-lang3.version>
-        <commons-text.version>1.12.0</commons-text.version>
-        <commons-io.version>2.16.1</commons-io.version>
-        <fhir-base.version>7.0.2-r8</fhir-base.version>
-        <fhir-structures.version>7.0.2-r5</fhir-structures.version>
-        <fhir-hl7-r4.version>6.3.3-r6</fhir-hl7-r4.version>
-        <fhir-hl7-utilities.version>6.3.3-r5</fhir-hl7-utilities.version>
+        <commons-lang3.version>3.17.0-r4</commons-lang3.version>
```

Comment:
adjust to r1 per my pull request (revision 1 of 3.17.0)

Reply:
Since artifacts `commons-io`, `commons-lang3` and `commons-text` are already included as dependencies within `hapi-fhir-base` , they are likely redundand in this `pom.xml`. The following warning message appeared in the terminal:

<img width="1353" alt="Screenshot 2024-09-13 at 11 33 24 AM" src="https://github.com/user-attachments/assets/72a2005f-7149-4046-a833-5a7c9fee808f">

After removing these dependencies and rebuilding `tinkar-core` everything worked as expected, and the warning message no longer appeared.

P.S. The screenshot was taken with the versions from the PR.

Reply:
Is this PR still needed? It looks like it fails to build even after the related PR in jpms-deps was merged in.

Reply:
The build is failing because the updated artifacts from the related [pull request](https://github.com/ikmdev/jpms-deps/pull/68) in the `jpms-deps` repository have not yet been published to Maven Central. As a result, `tinkar-core` cannot resolve these dependencies during the build process.

Here is a screenshot of the failed build:
<img width="1278" alt="Screenshot 2024-09-25 at 6 29 25 PM" src="https://github.com/user-attachments/assets/a6db9e88-76f6-4359-803d-da91c90600dc">

Below is the list of artifacts from the `jpms-deps` repository that need to be published to Maven Central. It’s critical to publish them in the specified order, as they have interdependencies:
1. commons-io (2.16.1-r1)
2. commons-lang3 (3.17.0-r1)
3. commons-text (1.12.0-r4)
4. guava (33.3.0-jre-r1)
5. hapi-fhir-base (7.0.2-r9)
6. hapi-fhir-caching-api (7.0.2-r1)
7. org.hl7.fhir.utilities (6.3.3-r8) - inside `jpms-deps/hapi-fhir-utilities` module
8. org.hl7.fhir.r4 (6.3.3-r9) - inside `jpms-deps/hapi-fhir-r4` module
9. hapi-fhir-structures-r4 (7.0.2-r8)

Publishing these artifacts in the correct order ensures that all dependencies are resolved properly during the build process.